src/main/java/org/example/UserService.java

```diff
@@ -0,0 +1,80 @@
+package org.example;
+
+import java.util.*;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class UserService {
+
+    public int getSumAgeByName(List<Client> clients, final String name) {
```

Comment:
For easier understanding add `javadoc` https://www.jetbrains.com/help/idea/working-with-code-documentation.html for these methods with an explanation what should they do